packages/website/src/pages/layout/Header.tsx

```diff
@@ -15,7 +14,7 @@ export default () => (
         propsMetadata={HeaderMetadata}
         demo={<HeaderDemo />}
         examples={{
-            modalVariant: <HeaderModalDemo grow title="Modal variant" />,
+            secondaryVariant: <HeaderSecondaryDemo grow title="Secondary variant" />,
```

Comment:
I have the impression that the import of the `HeaderSecondaryDemo` component is missing.

Reply:
yes i am dumb today